test/e2e/README.md

```diff
@@ -4,14 +4,12 @@ Automated end-to-end acceptance tests for the [wp-calypso](https://github.com/Au
 
 ## Table of contents
 
-- [Pre-requisites](#pre-requisites)
-  - [Install NodeJS](#install-nodejs)
-  - [Install dependencies](#install-dependencies)
-  - [Configuration](docs/config.md#configuration)
-    - [Test Configuration](docs/config.md#test-configuration)
-    - [Config Values](docs/config.md#config-values)
-    - [Standalone Environment Variables](docs/config.md#standalone-environment-variables)
-    - [CircleCI Environment Variables](docs/config.md#circleci-environment-variables)
+- [Setup](docs/setup.md)
+  - [software environment](https://github.com/worldomonation/wp-calypso/blob/update/e2e-documentation/test/e2e/docs/setup.md#software-environment)
+  - [configuration file](https://github.com/worldomonation/wp-calypso/blob/update/e2e-documentation/test/e2e/docs/setup.md#configuration)
+  - [environment variables](https://github.com/worldomonation/wp-calypso/blob/update/e2e-documentation/test/e2e/docs/setup.md#environment-variables)
+  - [branch naming](https://github.com/worldomonation/wp-calypso/blob/update/e2e-documentation/test/e2e/docs/setup.md#naming-branches)
+  - [set up user (optional)]()
```

Comment:
Should this have a link top something specific?

Reply:
Yes - I had not filled out the content of `set up user` yet and so I had left it blank. 

It is not something I've done yet, so my plan is to transplant the appropriate content of the FG page (PCYsg-jvX-p2) here in a subsequent push.

Reply:
I've decided to remove the subsection on `set up user` because it references some internal links, so would be much better suited to a consolidated page on FG.